docs/pages/es/functionality/management.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Lifecycle Management
+permalink: functionality/lifecycle-management
+type: functionality
+---
+
+# Gestión del ciclo de vida de la malla de servicio
+Meshery soporta varias [mallas de servicios diferentes](installation/adapters), de modo que puede aprender y comparar diferentes funcionalidades de malla de servicio. Una vez que haya seleccionado operar una mallas de servicios determinado, Meshery le ayudará con las actividades en curso.
+
+## Despliegue de una malla de servicios
+Meshery automatiza el aprovisionamiento de varias mallas de servicios, permitiendo a los usuarios seleccionar diferentes perfiles de configuración para admitir una variedad de modelos de despliegue.
+
+## Instalación de aplicaciones de muestra
+Comúnmente junto a cada malla de servicios hay una aplicación de muestra que facilita la demosatración del valor de la malla de servicios dada. Meshery permite que rapidamente despliegue la misma aplicación de muestra en diferentes mallas de servicios.

```

Comment:
typo: demostración

Reply:
everything else looks good, great work mate 👍

Reply:
Hi, @edwvilla, Do I need to change the type too?

Reply:
@edwvilla thank you very much for these reviews and your contributions. 👌